tests/Integration/inc/ThirdParty/Plugins/CDN/Cloudflare/displayServerPushingModeNotice.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace WP_Rocket\Tests\Integration\inc\ThirdParty\Plugins\CDN\Cloudflare;
+
+use Mockery;
+use Brain\Monkey\Functions;
+use WP_Rocket\Tests\Integration\TestCase;
+
+/**
+ * @covers \WP_Rocket\ThirdParty\Plugins\CDN\Cloudflare::display_server_pushing_mode_notice
+ *
+ * @group  AdminOnly
+ * @group  ThirdParty
+ */
+class Test_DisplayServerPushingModeNotice extends TestCase{
+
+	protected $rucss;
+    protected $combine_css;
+    private static $admin_user_id = 0;
+	private static $contributer_user_id = 0;
+
+	public static function set_up_before_class() {
+		parent::set_up_before_class();
+
+		$admin_role = get_role( 'administrator' );
+		$admin_role->add_cap( 'rocket_manage_options' );
+
+		self::$admin_user_id = static::factory()->user->create( [ 'role' => 'administrator' ] );
+		self::$contributer_user_id = static::factory()->user->create( [ 'role' => 'contributor' ] );
+	}
+
+	public function tear_down()
+	{
+		remove_filter('pre_get_rocket_option_remove_unused_css', [$this, 'rucss']);
+        remove_filter('pre_get_rocket_option_minify_concatenate_css', [$this, 'combine_css']);
+
+		parent::tear_down();
+	}
+
+	/**
+	 * @dataProvider configTestData
+	 */
+	public function testShouldDoAsExpected($config, $expected) {
+		
+		Functions\when( 'rocket_get_constant' )->alias(function($value) use($config) {
+			if ( $value == 'CLOUDFLARE_PLUGIN_DIR' || $value == 'CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE' ) {
+				return $config['server_push'];
+			}
+		});
```

Comment:
Instead of stubbing `rocket_get_constant`, you can use `$this->constants['CLOUDFLARE_HTTP2_SERVER_PUSH_ACTIVE'] = $config['server_push']` at the start of the test method, it will be automatically manage by the StubTrait that way